lib/matrix_sdk/protocols/cs.rb

```diff
@@ -767,6 +767,28 @@ def get_room_state_all(room_id, **params)
     request(:get, :client_r0, "/rooms/#{room_id}/state", query: query)
   end
 
+  # Retrieves number of events that happened just before and after the specified event
+  #
+  # @param room_id [MXID,String] The room to get events from.
+  # @param event_id [String] The event to get context around.
```

Comment:
Nit: Should probably be `[MXID,String]` here too, event IDs are also a kind of MXID